### PR TITLE
add my collection modules, asdf versions file #trivial

### PR DESCRIPTION
**Pull request by brainbicycle**
- adds some context modules for myCollection events
- adds a .tool-versions file to play nicely with asdf version manager

### CODE REVIEW

File: src/Schema/Values/ContextModule.ts [hunk 1]
```diff
@@ -66,6 +66,11 @@ export enum ContextModule {
   moreSeriesByThisArtist = "moreSeriesByThisArtist",
   myCollectionArtwork = "myCollectionArtwork",
   myCollectionHome = "myCollectionHome",
+  myCollectionArtworkAuctionResults = "myCollectionArtworkAuctionResults",
+  myCollectionArtworkHeader = "myCollectionArtworkHeader",
+  myCollectionArtworkMarketInsights = "myCollectionArtworkMarketInsights",
+  myCollectionArtworkRelatedArticles = "myCollectionArtworkRelatedArticles",
+  myCollectionArtworkSell = "myCollectionArtworkSell",
```
Comment (mikehrom) [inline src/Schema/Values/ContextModule.ts:73]: these are great. thinking though, do they need to be `myCollectionArtwork` specific if the screen owner specifies that much already?

Comment (mikehrom) [reply src/Schema/Values/ContextModule.ts:73]: we can do `marketInsights`, `auctionResults` etc.

Comment (brainbicycle) [reply src/Schema/Values/ContextModule.ts]: Updated, it looks like we have generic modules for most of these already just adding the marketInsights and artworkMetadata:

Existing contextModules:
https://github.com/artsy/cohesion/blob/135c07c4b9d9507424dbd09bf081fc15178b564f/src/Schema/Values/ContextModule.ts#L95
https://github.com/artsy/cohesion/blob/135c07c4b9d9507424dbd09bf081fc15178b564f/src/Schema/Values/ContextModule.ts#L29
https://github.com/artsy/cohesion/blob/135c07c4b9d9507424dbd09bf081fc15178b564f/src/Schema/Values/ContextModule.ts#L100

**Review by mikehrom — APPROVED** [review]

**Comment by artsyit** [comment]
:rocket: PR was released in [`v1.68.0`](https://github.com/artsy/cohesion/releases/tag/v1.68.0) :rocket:

**Pull request merged**